test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/cluster-autoscaler-binding.yaml

```diff
@@ -0,0 +1,15 @@
+# This is the role binding for the kubemark cluster autoscaler.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: cluster-autoscaler-view-binding
+  labels:
+    kubernetes.io/cluster-service: "true"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
This is fine for now.. but we should eventually have a proper role here (https://github.com/kubernetes/autoscaler/issues/383). Add a TODO maybe?

Reply:
Done.